xmake.lua

```diff
@@ -0,0 +1,32 @@
+set_config("ue4ssRoot", os.curdir())
+set_config("scriptsRoot", path.join(os.curdir(), "tools/xmakescripts"))
```

Comment:
QoL, consider `add_rules("plugin.vsxmake.autoupdate")` if we want to support auto-updating the VS project whenever changes are detected to the build lua files.